Fix cov-report tox environment run

* Cookiecutter version: master

```
(cookiecutter27) insspb@insspb-notebook:~/git/cookiecutter$ tox -e cov-report
cov-report create: /home/insspb/git/cookiecutter/.tox/cov-report
cov-report installdeps: -rtest_requirements.txt
cov-report installed: DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support,atomicwrites==1.3.0,attrs==19.3.0,configparser==4.0.2,contextlib2==0.6.0.post1,cookiecutter==1.7.0,coverage==5.1,freezegun==0.3.15,funcsigs==1.0.2,importlib-metadata==1.6.0,mock==3.0.5,more-itertools==5.0.0,packaging==20.3,pathlib2==2.3.5,pluggy==0.13.1,py==1.8.1,pyparsing==2.4.7,pytest==4.6.9,pytest-cov==2.8.1,pytest-mock==2.0.0,python-dateutil==2.8.1,scandir==1.10.0,six==1.14.0,wcwidth==0.1.9,zipp==1.2.0
cov-report run-test-pre: PYTHONHASHSEED='3635409391'
cov-report run-test: commands[0] | pytest --cov=cookiecutter --cov-report=term --cov-report=html
ImportError while loading conftest '/home/insspb/git/cookiecutter/tests/conftest.py'.
tests/conftest.py:11: in <module>
    from cookiecutter import utils
cookiecutter/utils.py:14: in <module>
    from cookiecutter.prompt import read_user_yes_no
cookiecutter/prompt.py:8: in <module>
    import click
E   ImportError: No module named click
ERROR: InvocationError for command /home/insspb/git/cookiecutter/.tox/cov-report/bin/pytest --cov=cookiecutter --cov-report=term --cov-report=html (exited with code 4)
___________________________________________________________________________________________________________ summary ___________________________________________________________________________________________________________
ERROR:   cov-report: commands failed
```

--- a/HISTORY.md
+++ b/HISTORY.md
@@ -18,7 +18,11 @@ Internal CI/CD changes:
 * Fixed Python 3.8 travis tests and setup.py message [@insspb](https://github.com/insspb) (#1295, #1297)
 * Travis builds extended with Windows setup for all supported python versions [@insspb](https://github.com/insspb) (#1300, #1301)
 * Update .travis.yml to be compatible with latest travis cfg specs [@luzfcb](https://github.com/luzfcb) (#1346)
-* Add new test to improve tests coverage [@amey589](https://github.com/amey589) (#1023)
+
+Changed tests:
+
+* Added new test to improve tests coverage [@amey589](https://github.com/amey589) (#1023)
+* Fixed `cov-report` tox invocation environment [@insspb](https://github.com/insspb) (#1350)
 
 Code style and docs changes:
 

--- a/HISTORY.md
+++ b/HISTORY.md
@@ -11,17 +11,17 @@ Important Changes:
 * 'future' library replaced with 'six' as a more lightweight python porting library [@asottile](https://github.com/asottile) (#941)
 * Added extension: Slugify template filter [@ppanero](https://github.com/ppanero) (#1336)
 * Added command line option: `--skip-if-file-exists`, allow to skip the existing files when doing `overwrite_if_exists`. [@chhsiao1981](https://github.com/chhsiao1981) (#1076)
+* Some packages versions limited to be compatible with python2.7 and python 3.5 [@insspb](https://github.com/insspb) (#1349)
 
-Internal CI/CD changes:
+Internal CI/CD and tests changes:
 
 * Coverage comment in future merge requests disabled [@ssbarnea](https://github.com/ssbarnea) (#1279)
 * Fixed Python 3.8 travis tests and setup.py message [@insspb](https://github.com/insspb) (#1295, #1297)
 * Travis builds extended with Windows setup for all supported python versions [@insspb](https://github.com/insspb) (#1300, #1301)
 * Update .travis.yml to be compatible with latest travis cfg specs [@luzfcb](https://github.com/luzfcb) (#1346)
-
-Changed tests:
-
 * Added new test to improve tests coverage [@amey589](https://github.com/amey589) (#1023)
+* Added missed coverage lines highlight to pytest-coverage report [@insspb](https://github.com/insspb) (#1352)
+* pytest-catchlog package removed from test_requirements, as now it is included in pytest [@insspb](https://github.com/insspb) (#1347)
 * Fixed `cov-report` tox invocation environment [@insspb](https://github.com/insspb) (#1350)
 
 Code style and docs changes:

--- a/setup.cfg
+++ b/setup.cfg
@@ -29,5 +29,4 @@ universal = 1
 
 [tool:pytest]
 testpaths = tests
-addopts = --cov=cookiecutter
-
+addopts = --cov-report term-missing --cov=cookiecutter

--- a/setup.py
+++ b/setup.py
@@ -25,14 +25,15 @@ with io.open('README.md', 'r', encoding='utf-8') as readme_file:
     readme = readme_file.read()
 
 requirements = [
-    'binaryornot>=0.2.0',
-    'jinja2>=2.7',
-    'click>=7.0',
-    'poyo>=0.1.0',
-    'jinja2-time>=0.1.0',
+    'binaryornot>=0.4.4',
+    'Jinja2<=2.11.0',
+    'click>=7.1.1',
+    'poyo>=0.5.0',
+    'jinja2-time>=0.2.0',
     'python-slugify>=4.0.0',
-    'requests>=2.18.0',
-    'six>=1.10',
+    'requests>=2.23.0',
+    'six>=1.14',
+    'MarkupSafe<2.0.0'
 ]
 
 if sys.argv[-1] == 'readme':

--- a/test_requirements.txt
+++ b/test_requirements.txt
@@ -1,5 +1,4 @@
 pytest
 pytest-cov
 pytest-mock
-pytest-catchlog
 freezegun

--- a/tox.ini
+++ b/tox.ini
@@ -34,4 +34,12 @@ usedevelop = false
 
 [testenv:cov-report]
 ; This coverage only report run. Only for local development. Not used at CI/CD
-commands = pytest --cov=cookiecutter --cov-report=term --cov-report=html
+passenv =
+    LC_ALL
+    LANG
+    HOME
+commands =
+    pip install -e .
+    pytest --cov=cookiecutter --cov-report=term --cov-report=html
+deps = -rtest_requirements.txt
+skip_install = true

